pkgs/cli/src/constants.zig

```diff
@@ -11,3 +11,6 @@ pub const DEFAULT_SERVER_STARTUP_TIMEOUT_MS: i64 = 120000;
 
 /// Default retry interval between connection attempts (in milliseconds)
 pub const DEFAULT_RETRY_INTERVAL_MS: u64 = 1000;
+
+/// Default path for the database
+pub const DEFAULT_DB_PATH: []const u8 = "./data";
```

Comment:
For Zig style, I don't think it need a `constants.zig`, each const should be put in their own module.

Reply:
It's a cli constant, I think it aligns with the current project style, but happy to address this based on comments from others.

Reply:
all cli consts can be aggregated just for pure convenience, so fine for now